internal/signing/channelaccounts/channel_accounts_model_test.go

```diff
@@ -0,0 +1,211 @@
+package channelaccounts
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/wallet-backend/internal/db"
+	"github.com/stellar/wallet-backend/internal/db/dbtest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func createChannelAccountFixture(t *testing.T, ctx context.Context, dbConnectionPool db.ConnectionPool, publicKey, encryptedPrivateKey string) {
```

Comment:
Maybe we can make this take an array of keypairs as param to avoid multiple rounds to the DB.